uber/site_sections/art_show_applications.py

```diff
@@ -100,8 +100,10 @@ def edit(self, session, message='', **params):
                 raise HTTPRedirect('..{}?id={}&message={}', return_to, app.id, message)
 
         receipt = session.refresh_receipt_and_model(app)
+        from uber.utils import get_static_file_path
```

Comment:
Is there a reason to do this here rather than up top?

Reply:
Oops! This was temporary while I was trying to debug what might be going on with get_static_file_path. I'll remove both this and the line that needed it.